gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py

```diff
@@ -3418,6 +3419,23 @@ def impl(context):
         if row:
             raise Exception("table %s has not finished expanding" % row[0])
 
+
+@then('table "{table_name}" {expansion_state} be marked as expanded')
+def impl(context, table_name, expansion_state):
+    dbname = 'postgres'
+    conn = dbconn.connect(dbconn.DbURL(dbname=dbname), unsetSearchPath=False)
+    try:
+        query = """select status from gpexpand.status_detail where fq_name='public.{0}'""".format(table_name)
+        result = dbconn.execSQLForSingleton(conn, query)
```

Comment:
I think, this change also should be described in commit message

Reply:
I don't think that such change should be described

Reply:
Done